.github/workflows/backport.yml

```diff
@@ -0,0 +1,20 @@
+name: Backport
+on:
+  pull_request:
+    types:
+      - closed
+      - labeled
+    paths-ignore:
+      - '_quarto.yml'
+      - 'quarto-materials/*'
+      - '**/.md'
+
+jobs:
+  backport:
+    runs-on: ubuntu-18.04
```

Comment:
Consider 20.04 or 22.04?

The suggested-to-be-included https://github.com/tibdex/backport/blob/main/.github/workflows/backport.yml has ubuntu-latest.

Reply:
Yes @eddelbuettel 100% copy from core

Will update ...

Reply:
... or perhaps leave as-is for compatibility with core

If this works out of the box, great. If it doesn't work out of the box (while core's does work) the first thing I'd want to do is remove all differerences between our config and core's ... so, perhaps best to leave them the same fro the get-go.

Reply:
Can 100% be argued both ways because what Core has clearly works yet may need an update....  

I am completely in support for "headache avoidance" so carbon-copy has a lot of merit.  

As, at least to me, has my hint to not encode ubuntu-18.04 in the year 2023 because they will withdraw for it at some point.  So a classic 'lose-lose', eventually.   But a win in the short-term.